lib/dgram.js

```diff
@@ -42,7 +42,10 @@ const {
   defaultTriggerAsyncIdScope,
   symbols: { async_id_symbol }
 } = require('internal/async_hooks');
+const { isUint32 } = require('internal/validators');
 const { UV_UDP_REUSEADDR } = process.binding('constants').os;
+const TTYWrap = process.binding('tty_wrap');
```

Comment:
I'm not fond of UDP requiring TTY. Maybe we should move that `guessHandleType` somewhere else?

Reply:
Maybe move the `guessHandleType ` into `lib/internal/util.js` or `lib/internal/net.js`?